vnext/PropertySheets/External/Microsoft.ReactNative.Uwp.CSharpApp.targets

```diff
@@ -18,11 +22,12 @@
     </PackageReference>
   </ItemGroup>
 
-   <!-- The props file for bundeling is not set up to be jsut defaults, it assumes to be run at the end of the project. -->
-  <Import Project="$(ReactNativeWindowsDir)PropertySheets\Bundle.props" />
-  <Import Project="$(ReactNativeWindowsDir)PropertySheets\Bundle.targets" />
+   <!-- The props file for bundling is not set up to be just defaults, it assumes to be run at the end of the project. -->
+  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\Bundle.props" />
```

Comment:
FYI directories variables in MSBuild often already include the trailing backslash (as is the case here). Adding the \ means it will look at foobar\\barbaz which is innocuous but unnecessary

Reply:
Unfortunately `ReactNativeWindowsDir` is a variable we explicitly set and expose in end user app project files, to give them the chance to override them (as we do within our in-repo projects to handle the mono-repo). Adding the `\` is an easy protection in case they forget.